### ISSUE TITLE
On history switch in multi view, go to Analyze data

### ISSUE THREAD

**Issue by afgane** [galaxyproject/galaxy#10479]
When in the multiple histories view, as a user, the intent of the `Switch to` button at the top of each history is to also start using the history. I'd suggest changing the button to `Switch to and use` where clicking on the button makes the selected history the active one and takes the user to the _Analyze Data_ view.

![image](https://user-images.githubusercontent.com/338642/96646289-ed295e00-12f9-11eb-8d8b-dabf94dc86d5.png)

**Comment by hexylena** [comment]
just a comment from a user perspective: I think this would bother me, I sometimes switch to a specific history, in order to drag datasets into it, before heading back to analyse. But maybe I just have a weird usage pattern here?

**Comment by mvdbeek** [comment]
Agreed, but ideally you could just drag datasets to any history, making that less necessary.

**Comment by astrovsky01** [comment]
Second @hexylena. This seems like it would mandate an order of operations in the multi-pane history, and just cause unnecessary pain points...

**Comment by afgane** [comment]
How about making it a two-part button, where `Switch to` keeps the user in the multi-history view and the `Use` portion takes the user directly to the Analyze Data view?
![image](https://user-images.githubusercontent.com/338642/99483899-3d153800-292d-11eb-88f1-06b4c6a355f6.png)
